### ISSUE TITLE
Allow user to specify the encoding when uploading a dataset

### ISSUE THREAD

**Issue by fpriyatna** [oeg-upm/mappingpedia-userinterface#17]
The default value should be UTF-8

**Comment by fpriyatna** [comment]
check with this file: https://www.dropbox.com/s/tzitw7n4hxwzfmw/mobile-age-iatreia.csv with distribution_encoding = Windows-1253

**Comment by ahmad88me** [comment]
@fpriyatna you already did that right?

**Comment by fpriyatna** [comment]
yes, you just need to specify the encoding in the parameter distribution_encoding
